static/app/views/eventsV2/table/queryField.tsx

```diff
@@ -202,6 +203,7 @@ class QueryField extends React.Component<Props> {
           fieldValue.function = [fieldValue.function[0], '', undefined, undefined];
         } else if (value.meta.parameters.length === 1) {
           fieldValue.function[2] = undefined;
+          fieldValue.function[3] = undefined;
```

Comment:
You need a condition to check I think
```
else if (value.meta.parameters.length === 2) {
    fieldValue.function[3] = undefined;
}
```

to account for count_miserable if it doesn't fulfill any of the conditions above.
To replicate, add a `count_if` with a `user` column and switch to `count_miserable`